tests/core/pyspec/eth2spec/test/helpers/genesis.py

```diff
@@ -28,6 +29,8 @@ def create_genesis_state(spec, validator_balances, activation_threshold):
 
     if spec.fork == ALTAIR:
         current_version = spec.ALTAIR_FORK_VERSION
+    elif spec.fork == MERGE:
+        current_version = spec.MERGE_FORK_VERSION
```

Comment:
Do we need to manually set `latest_execution_payload_header` in the generated state here? 

I suppose not because then the first non-genesis block looks like a transition block... but some other points lead me to believe we might be missing the execution_payload execution all together.

Reply:
The current merge transition code introduces the first payload on top of a zeroed `latest_execution_payload_header`, so it should work. I am not sure if/when this would change, or what should be the common case for testing. I prefer to just keep it zeroed for now though, and have later testing (PR #2385 ) change it after looking deeper into it.

Reply:
We may leave it for now. But technically it means that we do the transition during these tests and if we want to separate transition from post-merge test scenarios then we'd need to bootstrap the header in the state with some initial values. Initial value might has filled `block_hash` field with according value.